libs/eavmlib/src/esp.erl

```diff
@@ -287,3 +291,16 @@ rtc_slow_set_binary(Bin) when is_binary(Bin) ->
 -spec freq_hz() -> non_neg_integer().
 freq_hz() ->
     throw(nif_error).
+
+%%-----------------------------------------------------------------------------
+%% @param Interface the ESP32 network interface
+%% @returns The network MAC address of the specified interface
+%% @doc     Return the network MAC address of the specified interface.
+%%
+%%          The mac address is returned as a 6-byte binary, per the
+%%          IEEE 802 family of specifications.
+%% @end
+%%-----------------------------------------------------------------------------
+-spec get_mac(Interface :: interface()) -> mac().
+get_mac(_Interface) ->
+    throw(nif_error).
```

Comment:
We need erlang:nif_error(undefined) for dialyzer.

Reply:
fixed after rebase